contents/handbook/engineering/support-hero.md

```diff
@@ -58,11 +70,25 @@ As an engineer, when a question comes in your first instinct is to give them an
   - If they have a lot of volume and they're still on Postgres they should probably upgrade to Clickhouse
 - Sometimes questions will have been asked earlier in the User's Slack so it's worth searching through that if you're not sure.
 
-### Debugging helm charts
+#### Debugging helm charts
 
 Issues around deployments can be tricky to solve. It's useful to ask for context around:
 1. What is their `values.yml` file
 2. What guide they were following
 3. What is the output of `kubectl get pods -A`
 4. If any pod is not healthy, what is the output of `kubectl logs -n posthog <podname>`
 5. Helm chart version
+
+
+## Secondary on-call
+
+Every team has a Secondary on-call rotation. Unlike support hero, you are still expected to do feature work. During the week that you are on-call, you are responsible for prioritizing and solving escalated issues, sentry errors and alerts that happen within your team. It also means helping out the support hero where necessary.
```

Comment:
Note: For the deployments team we might want to do that's a full time support rather than trying to do feature work also, based the support load I see currently the whole team providing, which is only going up as we have more customers.

Reply:
@fuziontech ?

Reply:
I don't think we should make it full time yet. We'll pilot this as is and then we can figure out if this is a 20% thing, 50% thing, or 100% thing.